tests/contrib/operators/test_gcp_automl_operator_system.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+
+from tests.contrib.utils.base_gcp_system_test_case import (
+    SKIP_TEST_WARNING,
+    DagGcpSystemTestCase,
+)
+from tests.contrib.utils.gcp_authenticator import GCP_AUTOML_KEY
+
+
+@unittest.skipIf(DagGcpSystemTestCase.skip_check(GCP_AUTOML_KEY), SKIP_TEST_WARNING)
+class AutoMLDatasetOperationsSystemTest(DagGcpSystemTestCase):
+    def __init__(self, method_name="runTest"):
+        super().__init__(
+            method_name,
+            dag_id="example_automl_dataset",
+            require_local_executor=True,
+            gcp_key=GCP_AUTOML_KEY,
+        )
+
+    def test_run_example_dag(self):
+        self._run_dag()
+
+
+# @unittest.skipIf(DagGcpSystemTestCase.skip_check(GCP_AUTOML_KEY), SKIP_TEST_WARNING)
```

Comment:
One nit comment. I would not comment it all but add some extra SKIP_TEST that resolves to True by default. An environment variable that is never set in CI or something. For example we have 'CI' environment variable that is set to 'True' in CI environment. Some other tests are using 'TRAVIS' environment variable (but I am now changing it to more generic 'CI').

Reply:
This test last ~2h so I think it would be best to remove it. It could cause problems in case of exceeding any pipeline timeout.

Reply:
Yeah. I am all for removing it but by smart skipIf rather than by commenting out :) - so that you can run it locally still without changing (uncommenting) a code but by providing an environment variable. It's far too easy to uncomment it - if you want to run locally - and commit it.

Reply:
I have added `BaseGcpSystemTestCase.skip_long_test` method 👍🏼